src/clj/rems/css/styles.clj

```diff
@@ -117,12 +126,15 @@
                                           :border-left [[(u/px 10) :solid :white]]
                                           :border-bottom [[(u/px 20) :solid :transparent]]
                                           :border-right "none"}]
-    [:&.active {:background-color (util/get-theme-attribute :phase-color-active)
-                :border-color (util/get-theme-attribute :phase-color-active)
-                :color "#000"}]
-    [:&.completed {:background-color (util/get-theme-attribute :phase-color-completed)
-                   :border-color (util/get-theme-attribute :phase-color-completed)
-                   :color "#fff"}]]])
+    [:&.active (merge {:color (util/get-theme-attribute :phase-color-active :phase-color "#000")}
+                      (if-let [background (util/get-theme-attribute :phase-background-active)]
+                        {:background background}
+                        {:background-color (util/get-theme-attribute :phase-bgcolor-active)
+                         :border-color (util/get-theme-attribute
+                                        :phase-bgcolor-active)}))]
+    [:&.completed {:background-color (util/get-theme-attribute :phase-bgcolor-completed)
+                   :border-color (util/get-theme-attribute :phase-bgcolor-completed)
+                   :color (util/get-theme-attribute :phase-color-completed :phase-color "#fff")}]]])
```

Comment:
The fallback `"#fff"` is impossible to see on white background in the default theme. Using ` "#000"` as the fallback color for both active and completed phases would be more reliable.

Reply:
I changed the fallback colors and bgcolors to gray.